src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4852classannotations/InputClassAnnotations.java

```diff
@@ -1,35 +1,44 @@
 package com.google.checkstyle.test.chapter4formatting.rule4852classannotations;
 
+/**
+ * Test class for checking the rule 4852.
+ */
 public class InputClassAnnotations {
-    public @interface SomeAnnotation1 { }
+  /**
+   * Custom annotation 1.
+   */
```

Comment:
lets skip such comments.
Let minimize content that is not related to testing of chapter.

Reply:
We require javadocs for interfaces, classes, etc otherwise google config will give warnings